pom.xml

```diff
@@ -79,7 +79,7 @@
         <dependency>
             <groupId>io.swagger</groupId>
             <artifactId>swagger-parser</artifactId>
-            <version>1.0.8-SNAPSHOT</version>
+            <version>1.0.8</version>
```

Comment:
There is no 1.0.11 on Nexus. Should we put in there?
I've tried with 1.0.10 - build fails with NoClassDefFound (io.swagger.models.RefPaths) errors in tests.